sematic/abstract_function.py

```diff
@@ -1,27 +1,27 @@
 """
```

Comment:
Please keep an `abstract_calculator.py` in place which only raises an error saying `AbstractCalculator` has been renamed to `AbstractFunction`.
This way stray imports are informative. It is trivial to do and would be a good experience for any power or excentric users we might have.

Reply:
Hmm I would rather keep the codebase clean and avoid having to remove this file in the future. The import will fail anyway, just with a regular `ImportError`.